client/src/snippets.json

```diff
@@ -61,66 +79,66 @@
     "Select parameter with 3 options": {
         "prefix": "gx-select-3-param",
         "body": [
-            "<param name=\"$1\" type=\"select\" label=\"$2\" help=\"$3\">",
-            "    <option value=\"$4\">TODO: option $4</option>",
-            "    <option value=\"$5\">TODO: option $5</option>",
-            "    <option value=\"$6\">TODO: option $6</option>",
+            "<param name=\"$1\" type=\"select\" label=\"$2\" help=\"TODO\">",
+            "    <option value=\"$3\">TODO: $3</option>",
+            "    <option value=\"$4\">TODO: $4</option>",
+            "    <option value=\"$5\">TODO: $5</option>",
             "</param>"
         ],
         "description": "Select parameter with 3 options"
     },
     "Select parameter with 5 options": {
         "prefix": "gx-select-5-param",
         "body": [
-            "<param name=\"$1\" type=\"select\" label=\"$2\" help=\"$3\">",
-            "    <option value=\"$4\">TODO: option $4</option>",
-            "    <option value=\"$5\">TODO: option $5</option>",
-            "    <option value=\"$6\">TODO: option $6</option>",
-            "    <option value=\"$7\">TODO: option $7</option>",
-            "    <option value=\"$8\">TODO: option $8</option>",
+            "<param name=\"$1\" type=\"select\" label=\"$2\" help=\"TODO\">",
+            "    <option value=\"$3\">TODO: $3</option>",
+            "    <option value=\"$4\">TODO: $4</option>",
+            "    <option value=\"$5\">TODO: $5</option>",
+            "    <option value=\"$6\">TODO: $6</option>",
+            "    <option value=\"$7\">TODO: $7</option>",
             "</param>"
         ],
         "description": "Select parameter with 5 options"
     },
     "Boolean parameter": {
         "prefix": "gx-boolean-param",
         "body": [
-            "<param argument=\"$1\" type=\"boolean\" truevalue=\"$1\" falsevalue=\"$2\" label=\"$3\" help=\"$4\" />"
+            "<param argument=\"--$1\" type=\"boolean\" truevalue=\"$1\" falsevalue=\"$2\" label=\"$3\" help=\"$4\" />"
```

Comment:
truevalue and argument needs to have both the `--` --> please note that this can also be `-` for many tools ... But I agree that `--` is a better example.

Reply:
True, I have changed it to a default value in the placeholder. Now it should behave like this:
![sample-bool](https://user-images.githubusercontent.com/46503462/105610525-451ba900-5db0-11eb-94b2-28123fab44f8.gif)

So you can remove or add dashes as you desire and they will transfer to the `truevalue` attribute.

Reply:
Also now the help attributes have a default value too. This means that you have the opportunity to edit the help attribute contents while you are navigating with the next placeholder with the `TAB` key, but also you can just skip it and leave it as TODO so you can later write a longer or more detailed help description.